src/backend/controllers/CartController.js

```diff
@@ -1,5 +1,5 @@
-import { Response } from "miragejs";
```

Comment:
What was the need to make changes in the cartController? 

I can see it is just the change in the quotation marks and the same code is added.

Reply:
I have made one more change if you see closely in update cart handler, I have replaced +=1 with = action.payload according to my UI.